### PR TITLE
Adding the timestamp

### PR DESCRIPTION
**Pull request by thanashyam**
Timestamps are useful for the candidate to understand how recent the job posting is

### CODE REVIEW

**Review by AleTid5 — COMMENTED** [review]
Is excellent! 👏

File: src/components/Job/index.tsx [hunk 1]
```diff
@@ -92,6 +94,8 @@ export default ({
                 <Grid item xs={12}>
                   <Divider variant="middle" className={classes.spaceBottom} />
                   {positionName}
+                  <br />
+                  (<TimeAgo datetime={postDate} />)
```
Comment (AleTid5) [inline src/components/Job/index.tsx]: I think it would be much better if we put it inside a <i>i</i> tag, what do you think?

Comment (thanashyam) [reply src/components/Job/index.tsx]: Made the changes ⚡

Comment (AleTid5) [reply src/components/Job/index.tsx]: 🚀

**Review by AleTid5 — APPROVED** [review]
Excellent job!

**Pull request merged**